src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -20,6 +21,8 @@ public class FindCommand extends Command {
             + "Parameters: IC (must be a valid identity card number) \n"
             + "Example: " + COMMAND_WORD + " t1234567A";
 
+    private static Person foundPerson = null;
```

Comment:
@jiahui0309 I think ur right that this is violating some design principles. It'll also mean that if we add a new command (like `show`) then the `foundPerson` variable would need to be invalidated.